.github/workflows/build-docker-image.yml

```diff
@@ -57,8 +60,28 @@ jobs:
       # see https://github.com/testcontainers/moby-ryuk/pull/26
       # and https://github.com/testcontainers/moby-ryuk/pull/40
       - name: Cross Build for Linux and Windows
-        env:
-          RYUK_VERSION: ${{ github.event.release.tag_name || github.sha }}
-          IS_RELEASE: ${{ (github.event_name == 'release' && github.event.action == 'published' && 'yes') || 'no' }}
         run: |
           ./build-multiarch.sh testcontainers/ryuk:${RYUK_VERSION} ${IS_RELEASE}
+
+  run-images:
+    if: ${{ github.event_name == 'release' && github.event.action == 'published' }}
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, windows-2022]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Pull Ryuk
+        run: docker pull testcontainers/ryuk:${RYUK_VERSION}
+
+      - name: Run Ryuk on Linux
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
+        run: docker run -v /var/run/docker.sock:/var/run/docker.sock testcontainers/ryuk:${RYUK_VERSION}
+
+      - name: Run Ryuk on Windows
+        if: ${{ matrix.os == 'windows-2022' }}
+        run: docker run -v //./pipe/docker_engine://./pipe/docker_engine testcontainers/ryuk:${RYUK_VERSION}
+        continue-on-error: true
```

Comment:
Is this line still required?

Reply:
Probably we need to change it slightly, but we would like to keep something like this to test the built image. Due to the complexity and amount of combinations an error sneaked in. The `linux/amd64` and `windows/amd64` contained the ARM binary.

Reply:
Yes, I also think that the checks are required and make sense. I only asked for the "continue on error" line, which is at the very end of the workflow and doesn't have any effect (?).

Reply:
Yes, you are right. `continue-on-error` is not necessary. We will adjust it with the update that fixes the binaries. Unfortunately, for Windows more changes are necessary due to: https://github.com/actions/runner-images/issues/6688.